app/views/clients/rollup/_custom_services.html.haml

```diff
@@ -0,0 +1,13 @@
+:ruby
+  custom_services = @client.custom_services.
+    joins(enrollment: :client).
```

Comment:
Does this need to join client? I think we only need enrollment in order to do the merge with the visible enrollment scope.